README.md

```diff
@@ -1,4 +1,5 @@
-# Calchart
+# Calchart 
+[![calband](https://circleci.com/gh/calband/calchart-redesign.svg?style=shield)](https://circleci.com/gh/calband/calchart-redesign)
```

Comment:
Can we update this to be specifically for the `master` branch? Also can we update this link name from "calband" to "CircleCI runs"?